server.js

```diff
@@ -32,49 +32,47 @@ app.use(helmet());
 app.use(nonceMiddleware, helmet.contentSecurityPolicy({
   directives: {
     defaultSrc: ["'none'"],
-    baseUri: ['https://docs.helpscout.net'],
+    baseUri: ["'none'"],
     scriptSrc: [
       "'self'",
       "'strict-dynamic'",
       (req, res) => {
         return `'nonce-${res.locals.nonce}'`;
       },
-      'https://beacon-v2.helpscout.net',
-      'https://d12wqas9hcki3z.cloudfront.net',
-      'https://d33v4339jhl8k0.cloudfront.net',
+      'https://static.zdassets.com',
+      'https://ekr.zdassets.com',
+      'https://tidepool.zendesk.com',
+      'wss://tidepool.zendesk.com',
+      'wss://*.zopim.com'
```

Comment:
Not a bit deal, as it works either way, but we've decided to include trailing commas on the last elements of arrays and objects.

In our more up-to-date repos, like `viz`, this is enforced in our `esLint` settings, but we're in the process of upgrading a lot of the code in `blip` so we can't enforce it yet here, though we try to adhere to the same code formatting standards in any new code in `blip`

There are a few arrays in the CSP settings that should have the trailing commas added.